workers/stockList/init/buildPriceList.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/local/bin/node
+
+var fs = require('fs');
+var Promise = require('bluebird');
+var request = require('request');
+
+var api = 'https://query.yahooapis.com/v1/public/yql';
+var queryLeft = '?q=select%20*%20from%20yahoo.finance.quotes%20where%20symbol%20in("';
+var queryRight = '")&format=json&diagnostics=true&env=store://datatables.org/alltableswithkeys&callback=';
+
+
+var INPUT = __dirname + '/output/stocklist.json';
+var OUTPUT = __dirname + '/output/stocklistPrices.json';
+
+var readFile = Promise.promisify(fs.readFile);
+var writeFile = Promise.promisify(fs.writeFile);
+
+readFile(INPUT)
+  .then(function (json) {
+    return JSON.parse(json);
+  })
+  .then(function (stocks) {
+    console.log(new Date(), 'retrieving');
+    return getStockData(stocks);
+  })
+  .then(function (prices) {
+    console.log(new Date(), 'parse start');
+    var stockData = parseStockData(prices);
+    stockData = JSON.stringify(stockData, null, 4);
+    return writeFile(OUTPUT, stockData);
+  })
+  .catch(function (err) {
+    console.log('ERROR: ', err);
+  });
+
+function queryAPI(stocks) {
+  var api = 'https://query.yahooapis.com/v1/public/yql';
+
+  return new Promise(function (resolve, reject) {
+
+    var symbolQuery = stocks.map(function (stock) {
+      return stock.symbol;
+    }).join(',%20');
+
+    var query = api +
+      '?q=select%20*%20from%20yahoo.finance.quotes%20where%20symbol%20in("' +
+      symbolQuery +
```

Comment:
you defined queryLeft and queryRight above, but didn't use them here. Any reason why?